exchanges/stream/buffer/buffer.go

```diff
@@ -86,6 +92,11 @@ func (w *Orderbook) Update(u *Update) error {
 			u.Asset)
 	}
 
+	// out of order update ID can be skipped
+	if w.updateIDProgression && u.UpdateID <= book.updateID {
+		return nil
+	}
```

Comment:
Please add a test scenario for this

Reply:
What are your thoughts on a `if verbose { log("out of order websocket update received") }` ?